man/systemd-analyze.xml

```diff
@@ -792,6 +792,59 @@ Service b@0.service not loaded, b.socket cannot be started.
         as well and its default value is 100.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--security-policy=<replaceable>PATH</replaceable></option></term>
+
+        <listitem><para>With <command>security</command>, allow the user to define a custom set of
+        requirements formatted as a .json file against which to compare the specified unit file(s)
+        and determine their overall exposure level to security threats.</para></listitem>
+      </varlistentry>
```

Comment:
I guess we should define the accepted IDs here, given they are fixed. Something like this maybe:

```
        <table>
          <title>Accepted Directives</title>

          <tgroup cols='1'>
            <colspec colname='directive' />
            <thead>
              <row>
                <entry>Directive</entry>
              </row>
            </thead>
            <tbody>
              <row>
                <entry>PrivateDevices</entry>
              </row>
              <row>
                <entry>PrivateMounts</entry>
              </row>
              ...
            </tbody>
          </tgroup>
        </table>
```